Proofs/AES-GCM/GCMGmultV8Sym.lean

```diff
@@ -5,21 +5,74 @@ Author(s): Alex Keizer
 -/
 import Tests.«AES-GCM».GCMGmultV8Program
 import Tactics.Sym
+import Tactics.Aggregate
 import Tactics.StepThms
+import Tactics.CSE
+import Arm.Memory.SeparateAutomation
+import Arm.Syntax
 
 namespace GCMGmultV8Program
+open ArmStateNotation
 
 #genStepEqTheorems gcm_gmult_v8_program
 
+/-
+xxx: GCMGmultV8 Helem HTable
+-/
+
+set_option pp.deepTerms false in
+set_option pp.deepTerms.threshold 20 in
+-- set_option trace.simp_mem.info true in
 theorem gcm_gmult_v8_program_run_27 (s0 sf : ArmState)
     (h_s0_program : s0.program = gcm_gmult_v8_program)
     (h_s0_err : read_err s0 = .None)
     (h_s0_pc : read_pc s0 = gcm_gmult_v8_program.min)
     (h_s0_sp_aligned : CheckSPAlignment s0)
+    (h_Helem : Helem = s0[read_gpr 64 0#5 s0, 16])
+    (h_HTable : HTable = s0[read_gpr 64 1#5 s0, 256])
+    (h_mem_sep : Memory.Region.pairwiseSeparate
+                 [(read_gpr 64 0#5 s0, 16),
+                  (read_gpr 64 1#5 s0, 256)])
     (h_run : sf = run gcm_gmult_v8_program.length s0) :
-    read_err sf = .None := by
+    read_err sf = .None ∧
```

Comment:
Worthwhile to add conclusions for `sf.program`, `read_pc sf` and `CheckSPAlignment sf`?

Reply:
Done. I added more frame conditions too. Thanks!